Corvus.Extensions.Samples/TaskExSample.dib

```diff
@@ -0,0 +1,127 @@
+#!markdown
+
+## TaskEx - Sample notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Extensions` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Extensions, 1.1.4"
+
+#!markdown
+
+Add using statement for `Corvus.Extensions` namespace
+
+#!csharp
+
+using System.IO;
+using Corvus.Extensions;
+using Corvus.Extensions.Tasks;
+
+#!markdown
+
+### WhenAllMany()
+
+This method flattens the outputs from `Task.WhenAll`
```

Comment:
I'm not sure if `Task.WhenAll` really belongs in this description. It feels more like a description of the implementation than the purpose. (I only just noticed that it's also in the `<summary>` of the XML doc comment... Hum.)

The thing is, this both expands and flattens. It's just like `SelectMany` so maybe we should look at that:

>Projects each element of a sequence to an [IEnumerable<T>](https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.ienumerable-1?view=net-6.0) and flattens the resulting sequences into one sequence.

I wonder if we should say: "Projects each element of a sequence to a `Task<IEnumerable<T>>` and flattens the sequences produced by the resulting tasks into one `Task<IList<T>>`.

Or we could be more precise:

"Passes the elements of a sequence to a callback that projects each element to a `Task<IEnumerable<T>>` and flattens the sequences produced by the resulting tasks into one `Task<IList<T>>`.

Reply:
Thanks, I've used your more precise version.